roofit/roofitcore/inc/RooVectorDataStore.h

```diff
@@ -293,7 +293,7 @@ class RooVectorDataStore : public RooAbsDataStore {
       auto beg = std::min(_vec.cbegin() + first, _vec.cend());
       auto end = std::min(_vec.cbegin() + last,  _vec.cend());
 
-      return RooSpan<const double>(beg, end);
+      return RooSpan<const double>(&*beg, std::distance(beg, end));
```

Comment:
```suggestion
      return RooSpan<const double>(beg.data(), std::distance(beg, end));
```

Better use `data()` to get the underlying pointer for vectors (also in the other lines)